sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/module-info.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.resourcemanager.resources {
+    requires transitive com.azure.core.management;
+    requires com.github.spotbugs.annotations;
```

Comment:
Why do we need spotbugs annotations in this module?

Reply:
This is due to the [suppression for spotbugs](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/dag/TaskGroup.java#L125).

Reply:
@xccc-msft Spotbugs suppression should be done in `spotbugs-exclude.xml` file:
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

Reply:
Updated.